Material_Dialog/entry/src/main/ets/MainAbility/pages/index.ets

```diff
@@ -0,0 +1,292 @@
+@CustomDialog
```

Comment:
instead of adding components in index file create separate files for different components and use model class for properties handling.
Create a library for the components